src/SmartComponents/Recs/Details.js

```diff
@@ -168,7 +169,9 @@ const OverviewDetails = ({ match, fetchRuleAck, fetchTopics, fetchSystem, fetchR
                             <PageHeaderTitle title={<React.Fragment><RuleLabels rule={rule} />{rule.description}</React.Fragment>} />
                             <p>{intl.formatMessage(
                                 messages.rulesDetailsPubishdate, { date: <DateFormat date={new Date(rule.publish_date)} type="onlyDate" /> }
-                            )}</p>
+                            )}
+                            &nbsp;&nbsp;&nbsp;<Label color="blue">{rule.category.name}</Label>
```

Comment:
lets use `--pf-global--spacer--sm` on the label for the margin (or maybe md) rather than padding with &nbsp;

Reply:
Done.  Much more profeshonal looking too!